dxr/static/templates/path_line.html

```diff
@@ -1,4 +1,10 @@
 {%- if not is_first_or_only -%}
 <span class="path-separator">/</span>
 {%- endif -%}
-<a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>{{ display_path }}</a>
\ No newline at end of file
+{%- if not is_binary and not is_dir -%}
+<a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>
+{%- endif -%}
+    {{ display_path }}
+{%- if not is_binary -%}
```

Comment:
My mistake! We'd have an extra closing tag for all of the path crumbs leading up to the file.